src/components/Details/index.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable camelcase */
+import React from 'react';
```

Comment:
- [ ] Kindly avoid disabling the linter since it enforces rules for quality code. Instead, you should fix the raised issues.

Reply:
Used camel case issue resolved

Reply:
However to access the Capital i need to disable the camel lint because that the only way to fetch that data, so i wont be fetching the Capital now that i eneable camelCasing